system/jlib/jdebug.cpp

```diff
@@ -1650,8 +1651,18 @@ class CExtendedStats  // Disk network and cpu stats
 
     StringBuffer ifname;
 
+#ifdef __linux__
+    UnsignedArray majmin;
+#endif
+
     int isdisk(unsigned int major, unsigned int minor)
     {
+#ifdef __linux__
```

Comment:
The old code is there in case the lsblk command doesn't work or is not there or whatever reason.
getLine() is called if not WIN32 so maybe also APPLE but I'm not sure that would work ??

Reply:
ok, yes ok that makes sense,
but isDisk() is only ever called under Linux isn't it, because it's relying on /proc/diskstats, which is linux only ?

I'd remove the #ifdef __linux__ here, because this is a generic function with no linux functionality, but it will only be called/used where appropriate.